cli/lightning-cli.c

```diff
@@ -286,11 +304,11 @@ static bool is_literal(const char *arg)
 }
 
 static void add_input(char **cmd, const char *input,
-		      int i, int argc)
+		      int i, int argc, enum escape escape)
 {
```

Comment:
Thank you. If we make this an experimental feature, then we would not be able to use it to finish PR #4821 until much later...just something to keep in mind.

The name `info` was chosen because in GraphQL it must be the same as the field name in the result set. It would be strange to have `getinfo` as the name in the result set.

Reply:
>Thank you. If we make this an experimental feature, then we would not be able to use it to finish PR #4821 until much later...just something to keep in mind.

I'm not able to see any information on how you want to try to use this PR in the #4821, but anyway. I'm missing some things.

>The name info was chosen because in GraphQL it must be the same as the field name in the result set. It would be strange to have getinfo as the name in the result set.

It looks like a fork of opinion, but it makes sense.

Thanks

Reply:
> > Thank you. If we make this an experimental feature, then we would not be able to use it to finish PR #4821 until much later...just something to keep in mind.
> 
> I'm not able to see any information on how you want to try to use this PR in the #4821, but anyway. I'm missing some things.

The topology plugin currently calls `listpeers` and needs some channel info from the call, to know if each peer has an appropriate channel or not. For that reason, deprecating channels from `listpeers` breaks the topology plugin. To build the fix on top of this PR, the topology plugin can just call `graphql {peers{id,connected,channels{state}}}` and get the exact data it cares about. But ok, there are other ways to fix the topology plugin, even if it costs more RPC calls... it was just an idea to take advantage of this PR as a base.